bibharvest/bibfilter_oaipos2inspire.py

```diff
@@ -108,6 +108,10 @@ def main(args):
         for link in links:
             url = urllib.quote(link['href'], safe=":/")
             if url.endswith('.pdf'):
+                # relative URLs
+                if url.startswith('/'):
+                    protocol, _, domain, _ = base_url.split('/', 3)
+                    url = protocol + '//' + domain + url
```

Comment:
Looks good. Alternatively you could use https://docs.python.org/release/2.6.9/library/urlparse.html#urlparse.urljoin which seems designed for this use case as well.